<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DebugOutput</class>
 <widget class="QWidget" name="DebugOutput">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>925</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Debug Output</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="lineWrapMode">
      <enum>QTextEdit::NoWrap</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="copyPushButton">
       <property name="text">
        <string>Copy to Clipboard</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="autoScrollCheckBox">
       <property name="text">
        <string>Auto Scroll</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="onTopCheckBox">
       <property name="text">
        <string>Keep Window On Top</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="commitLabel">
       <property name="text">
        <string>Commit:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="commitLineEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="hashLabel">
       <property name="text">
        <string>Hash:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="hashLineEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
